osu.Game.Rulesets.Catch/Beatmaps/CatchBeatmapProcessor.cs

```diff
@@ -100,7 +100,7 @@ private void initialiseHyperDash(List<CatchHitObject> objects)
                 CatchHitObject nextObject = objectWithDroplets[i + 1];
 
                 int thisDirection = nextObject.X > currentObject.X ? 1 : -1;
-                double timeToNext = nextObject.StartTime - currentObject.StartTime;
+                double timeToNext = nextObject.StartTime - currentObject.StartTime - 1000f / 60f / 4; // 4 frames of grace time, taken from osu-stable
```

Comment:
this is 1/4 of a frame, not 4 frames. can you confirm where you're referencing to get this value?

Reply:
Oh, my bad. InitializeHyperDash in HitObjectManagerFruits substracts 4.1666 from timeToNext. I rushed to get this PR ready since I had to leave home. The value is correct, the comment isn't.